src/PureBlazor.Components/Buttons/ButtonProps.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.AspNetCore.Components;
 
-namespace PureBlazor.Components.Buttons;
+namespace PureBlazor.Components;
```

Comment:
We definitely want this to match the folder structure - so the namespace should be `PureBlazor.Components.Buttons`

Reply:
I'm confused, so all files must be updated with this convention?

For example:
PureToast.razor change it to @namespace PureBlazor.Components.Alerts;
SkipToContent.razor change it to @namespace PureBlazor.Components.Accessibility;
PureCode.razor change it to @namespace PureBlazor.Components.Typography;
PureCard.razor change it to @namespace PureBlazor.Components.Layout;

And there are more cases like the one mentioned here

Reply:
Any regular .razor file automatically is given the namespace based on the file/folder location. 

So for instance, by being in
`PureBlazor.Components\Buttons`, the file has the namespace `PureBlazor.Components.Buttons`

The only time we need to declare the namespace is for when we split out the code behind files. 

In the original version of Makani, I thought it would be better to force all components to show up under a single namespace, so I didn't need to import each individual folder. I no longer think this is an issue, especially with using the global Imports file. 

Let me know if this helps clear up the confusion...

Reply:
I have reverted the namespace convention for the button components.
Next week we can discuss some namespace naming rules